src/openiddict-test/project.json

```diff
@@ -32,7 +31,9 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0",
     "Microsoft.Extensions.Logging.Console": "1.0.0",
     "Microsoft.Extensions.Logging.Debug": "1.0.0",
-    "Microsoft.NETCore.Platforms": "1.0.1"
+    "Microsoft.NETCore.Platforms": "1.0.1",
+    "AspNet.Security.OAuth.Validation": "1.0.0-alpha3-0181",
```

Comment:
Curious: any reason to move to the nightly builds instead of using the latest official release?